src/sentry/scim/endpoints/members.py

```diff
@@ -565,11 +565,12 @@ def post(self, request: Request, organization) -> Response:
                     organization=organization, email=result["email"], role=result["role"]
                 )
 
+                region_outbox = None
                 if member_query.exists():
                     member = member_query.first()
                     if member.token_expired:
                         member.regenerate_token()
-
+                        region_outbox = member.save()
```

Comment:
We've never saved the regenerated token here. So I added `save()` here, and updated the appropriate test to check if the token is not expired.

Reply:
Nice find :clap: